contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -191,11 +215,24 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   function _calculatePaymentAmount(
```

Comment:
can we add some natspec comments here? would be nice to have an explanation for `isExecution` and `txCallData` (the others too)

Reply:
Done